Model/QueueHandler/VarnishUrlPurger.php

```diff
@@ -146,6 +100,28 @@ protected function getQueueProcessType(): string
         return self::PROCESS_TYPE;
     }
 
+    /**
+     *
+     * @param string $url
+     * @return void
+     */
+    private function createRequest(string $url): void
+    {
+        $client = $this->clientFactory->create($this->loop);
+        $request = $client->request('GET', $url, $this->buildHeaders());
+        $request->on('response', function (Response $response) use ($url) {
```

Comment:
Maybe it would reasonable to attach some callback on error? End user maybe should be informed about error and url should be pushed to the end of the queue?